tools/webidl_binder.py

```diff
@@ -515,9 +517,10 @@ def render_function(class_name, func_name, sigs, return_type, non_pointer,
   body += '  %s%s(%s)%s;\n' % (call_prefix, '_' + c_names[max_args], ', '.join(pre_arg + args), call_postfix)
   if cache:
     body += '  ' + cache + '\n'
-  mid_js.append(r'''%sfunction%s(%s) {
+  mid_js.append(r'''function%s(%s) {
 %s
-};''' % (CONSTRUCTOR_CLOSURE_SUPPRESSIONS, (' ' + func_name) if constructor else '', ', '.join(args), body[:-1]))
```

Comment:
Why are the closure suppressions not needed here?

Reply:
They are still needed and used everywhere, I just put them on the preceding line to make things more readable in the output.

Reply:
Its now the callers of `render_function` that add `CONSTRUCTOR_CLOSURE_SUPPRESSIONS` rather than `render_function` itself.

Reply:
I wonder if we should checkin a copy of the generated glue code so that we can see the changes to output?

Reply:
I see, sgtm.

Not sure it's worth checking in the code here. We do have good test coverage including closure.